apps/backend/src/modules/projects/project.repository.ts

```diff
@@ -27,8 +28,13 @@ class ProjectRepository implements Repository {
 		return ProjectEntity.initialize(user);
 	}
 
-	public delete(): ReturnType<Repository["delete"]> {
-		return Promise.resolve(true);
+	public async delete(id: number): Promise<boolean> {
+		const deletedRowsCount = await this.projectModel
+			.query()
+			.deleteById(id)
+			.execute();
+
+		return deletedRowsCount > NOTHING_DELETED_COUNT;
```

Comment:
let's just wrap it with Boolean
```suggestion
		return Boolean(deletedRowsCount);
```

Reply:
That's genious ;)